lib/robots/dor_repo/gis_delivery/load_raster.rb

```diff
@@ -14,14 +14,7 @@ def perform_work
           rootdir = GisRobotSuite.locate_druid_path bare_druid, type: :workspace
 
           # determine whether we have a Raster to load
```

Comment:
this comment seems like it could use a tweak with the switch from `descMetadata.xml` to the method taking the cocina object

Reply:
Why?

Reply:
oh, hmm... i guess i was thinking of that comment as being about loading the `descMetadata.xml`, which is no longer happening.  but on further thought, i guess it was more about the loading to geoserver that happens after that, not the file read call that was right below it?

so maybe it was a slightly misplaced comment before, and now it's well placed, and i was just confused by my (wrong?) assumption that the old comment was a good one?

ok, yeah, sold on the comment as-is.